centreon/src/App/MonitoringConfiguration/Infrastructure/Dbal/DbalCommandRepository.php

```diff
@@ -164,6 +143,23 @@ public function add(Command $command): void
         $this->setId($command, new CommandId($id));
     }
 
+    /**
+     * @return array<string>
+     */
+    public static function getSelectColumns(string $alias = 'cm'): array
+    {
+        return [
+            "{$alias}.command_id AS cm_command_id",
```

Comment:
I'm not sure to understand the reason for columns alias cm_xxx, is it a personal preference ?